samples/bookinfo/src/ratings/ratings.js

```diff
@@ -33,6 +35,34 @@ if (process.env.SERVICE_VERSION === 'v2') {
   }
 }
 
+dispatcher.onPost(/^\/ratings\/[0-9]*/, function (req, res) {
+  var productIdStr = req.url.split('/').pop()
+  var productId = parseInt(productIdStr)
+  var ratings = {}
+
+  if (Number.isNaN(productId)) {
+    res.writeHead(400, {'Content-type': 'application/json'})
+    res.end(JSON.stringify({error: 'please provide numeric product ID'}))
+    return
+  }
+
+  try {
+    ratings = JSON.parse(req.body)
+  } catch (error) {
+    res.writeHead(400, {'Content-type': 'application/json'})
+    res.end(JSON.stringify({error: 'please provide valid ratings JSON'}))
+    return
+  }
+
+  if (process.env.SERVICE_VERSION === 'v2') {
```

Comment:
I'd add a comment on why this is required.

Reply:
Addressed in 6b4dac8e532d6b1ff5f56596ad99c33f2abf42ef